apps/web-giddh/src/assets/locale/mr.json

```diff
@@ -528,6 +528,8 @@
     "app_bulk_export": "मोठ्या प्रमाणात निर्यात",
     "app_pin_code": "पिन कोड",
     "app_enter_pin_code": "पिन कोड प्रविष्ट करा",
+    "app_zip_code": "पिनकोड",
```

Comment:
use correct translation

Reply:
This correct I have checked it again on google translation and chatGPT too